### PR TITLE
Fix svg errors

### PR DESCRIPTION
**Pull request by smyzin**
- fixed errors in Chrome + warnings in Firefox

### CODE REVIEW

**Review by MixailE — APPROVED** [review]

File: src/icons/BurnerWalletSeal.tsx [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+)
+const pathClass = classnames(strokeWidth('stroke-2'), stroke('stroke-tertiary'))
+
+export default function () {
+  return (
+    <svg
+      width="114"
+      height="103"
+      viewBox="0 0 114 103"
+      fill="none"
```
Comment (T-Damer) [inline src/icons/BurnerWalletSeal.tsx:16]: Let's remove the fill?

File: src/components/BurnerWalletCard/BurnerWalletHeaderLogo.tsx [hunk 2]
```diff
@@ -1,17 +1,20 @@
...
 
 export default function () {
   return (
     <>
       <div className={displayFrom('md')}>
         <BurnerWalletLogo />
       </div>
-      <div className={smallLogo}>
-        <BurnerWalletLogo small />
+      <div className={displayTo('md')}>
```
Comment (T-Damer) [inline src/components/BurnerWalletCard/BurnerWalletHeaderLogo.tsx:11]: Wait, so the whole block of `wallets` will be displayed only before `md`?

File: src/icons/BurnerWalletSeal.tsx (hunk 1 above)
Comment (smyzin) [reply src/icons/BurnerWalletSeal.tsx:16]: Done

File: src/components/BurnerWalletCard/BurnerWalletHeaderLogo.tsx (hunk 2 above)
Comment (smyzin) [reply src/components/BurnerWalletCard/BurnerWalletHeaderLogo.tsx:11]: No, only Logo part

**Pull request merged**